CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Breaking Changes
 
 * Updated `iso8601` dependency to 0.3.0
+* Added a new default feature named `tls` that depends on the (default) tls support in [reqwest].
```

Comment:
The real feature you're adding is making TLS optional, so maybe this would be better:

```suggestion
* Added a new default feature `tls` that can be disabled to turn off [reqwest]'s TLS support.
```